test/distributed/optim/test_zero_redundancy_optimizer.py

```diff
@@ -61,8 +61,11 @@ def tearDown(self):
             pass
 
     def dist_init(self, rank, world_size=-1):
-        store = dist.FileStore(self.file_name, self.world_size if world_size < 1 else world_size)
-        return dist.init_process_group(backend=BACKEND, store=store, rank=rank, world_size=self.world_size)
+        if (world_size < 1):
```

Comment:
The idea here is to make the `world_size` consistent between `store` and `dist.init_process_group()`. Previously, it is possible that the `world_size` passed into `dist_init()` is at least `1` but is inconsistent with `self.world_size`, leading to a discrepancy between `store` and `dist.init_process_group()`.